Specs/Scene/ModelExperimental/ModelExperimentalNodeSpec.js

```diff
@@ -217,4 +219,47 @@ describe("Scene/ModelExperimental/ModelExperimentalNode", function () {
     expect(Matrix4.equals(node.transform, newTransform)).toBe(true);
     expect(Matrix4.equals(node.originalTransform, transform)).toBe(true);
   });
+
+  it("updateTransforms updates matrices", function () {
+    const node = new ModelExperimentalNode({
+      node: mockNode,
+      transform: transform,
+      transformToRoot: transformToRoot,
+      sceneGraph: mockSceneGraph,
+      children: [0],
+    });
+
+    const axisCorrected = ModelExperimentalUtility.correctModelMatrix(
+      Matrix4.IDENTITY,
+      mockSceneGraph.components.upAxis,
+      mockSceneGraph.components.forwardAxis,
+      new Matrix4()
+    );
+
+    expect(Matrix4.equals(node.computedTransform, Matrix4.IDENTITY)).toBe(true);
```

Comment:
Can we add checks for `node.computedTransform` and `node.axisCorrectedTransform` to the `verifyTransforms` function for this spec? Then we can reuse `verifyTransforms` here instead of individual `expect` checks